docs/content/preview/tutorials/google/google-vertex-ai.md

```diff
@@ -0,0 +1,163 @@
+---
+In this tutorial, we'll walk you through the steps to build generative AI applications using Google Vertex AI and YugabyteDB.
+
+The application provides lodging recommendations for travelers visiting San Francisco.
+
+![YugaLodgings Application](/images/tutorials/google/google-vertex-ai/yugalodgings-main.png "YugaLodgings Application")
+
+Using Google Vertex AI, text embeddings (a vectorized representation of the data) are generated for each listing description and stored in YugabyteDB, using the PostgreSQL `pgvector` extension. The user's prompt is similarly converted to text embeddings using Google Vertex AI and subsequently used to execute a similarity search in YugabyteDB, finding properties with descriptions related to the user's prompt.
+
+## Prerequisites
+* A Google Cloud account with appropriate permissions
+* A YugabyteDB cluster of version [2.19.2 or later](https://download.yugabyte.com/)
+* Install Node.js v18+
+* Install [Docker](https://docs.docker.com/get-docker/)
+
+## Initializing the Project
+
+1. Clone the repository.
+  ```shell
+    git clone https://github.com/YugabyteDB-Samples/yugabytedb-google-vertexai-lodging-service.git
+  ```
+2. Install the application dependencies.
+   ```shell
+     cd {project_directory}/backend
+     npm install
+
+
+     cd {project_directory}/frontend
+     npm install
+   ```
+3. Configure the application environment variables in `{project_directory/backend/.env}`.
+
+## Start YugabyteDB and Load the Sample Dataset
+
+1. Start a YugabyteDB isntance of version 2.19.2 or later:
+```shell
+mkdir ~/yb_docker_data
+
+docker network create custom-network
+
+docker run -d --name yugabytedb_node1 --net custom-network \
+   -p 15433:15433 -p 7001:7000 -p 9001:9000 -p 5433:5433 \
+   -v ~/yb_docker_data/node1:/home/yugabyte/yb_data --restart unless-stopped \
+   yugabytedb/yugabyte:2.19.2.0-b121 \
+   bin/yugabyted start \
+   --base_dir=/home/yugabyte/yb_data --daemon=false
+
+docker run -d --name yugabytedb_node2 --net custom-network \
+   -p 15434:15433 -p 7002:7000 -p 9002:9000 -p 5434:5433 \
+   -v ~/yb_docker_data/node2:/home/yugabyte/yb_data --restart unless-stopped \
+   yugabytedb/yugabyte:2.19.2.0-b121 \
+   bin/yugabyted start --join=yugabytedb_node1 \
+   --base_dir=/home/yugabyte/yb_data --daemon=false
+  
+docker run -d --name yugabytedb_node3 --net custom-network \
+   -p 15435:15433 -p 7003:7000 -p 9003:9000 -p 5435:5433 \
+   -v ~/yb_docker_data/node3:/home/yugabyte/yb_data --restart unless-stopped \
+   yugabytedb/yugabyte:2.19.2.0-b121 \
+   bin/yugabyted start --join=yugabytedb_node1 \
+   --base_dir=/home/yugabyte/yb_data --daemon=false
+```
+The database connectivity settings are provided in the `{project_dir}/backend/.env` file and do not need to be changed if you started the cluster with the command above.
+
+1. Copy the Airbnb schema and data to the first node's container:
+   ```shell
+    docker cp {project_dir}/sql/0_airbnb_listings.sql yugabytedb-node1:/home
```

Comment:
The instructions from this and the next bullet item will fail because you are referencing "yugabytedb-node1," but the container's name differs ("yugabytedb_node1"). 

My suggestion is to name the containers as "yugabytedb-node{number}". The "_" symbol no longer works in the container's name in the latest versions of YugabyteDB.

Reply:
Thanks, Dwight updated accordingly.